index.html

```diff
@@ -0,0 +1,1753 @@
+              </path>
+            </g>
+          </svg>
+        </a>
+        <div class="col p-0">
+          <div class="row justify-content-end align-content-center text-uppercase font-weight-bold pr-0">
+            <a class="text-decoration-none text-white px-2 pt-2" href="#">Sign in</a>
+            <a class="text-decoration-none px-2 pt-2" style="background: white; color:#f72210; " href="#">Subscribe <i
+                class="fas fa-chevron-right pl-1"></i></a>
+            <button class="navbar-toggler p-0 ml-2" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
+              <span class="navbar-toggler-icon pt-2" style="color: white;"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="collapsingNavbar">
+              <ul class="style navbar-nav float-right  mt-3" style="margin-right: -1em; background: white;">
```

Comment:
**[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.